conbench/tests/api/_expected_docs.py

```diff
@@ -305,6 +305,63 @@
                 },
                 "description": "OK",
             },
+            "CompareBenchmarkResult": {
```

Comment:
I'm assuming it's okay that some of these are reordered; just an artifact of how swagger works

Reply:
Yep.